CRM/MembershipExtras/Hook/Post/MembershipPayment.php

```diff
@@ -38,9 +38,26 @@ public function __construct($operation, $objectId, CRM_Member_DAO_MembershipPaym
   public function postProcess() {
     if ($this->operation == 'create') {
       $this->fixRecurringLineItemMembershipReferences();
+      $this->recalculateMembershipStatus();
     }
   }
 
+  /**
+   * Recaalculates the status of the related membership.
+   *
+   * Recalculates the statos of the related membership to check if payment plan }
```

Comment:
some typo : statos  => status


and there is an extra }

Reply:
Fixed!